### PR TITLE
Updates to instructions

### PR DESCRIPTION
**Pull request by alexellis**
* Included FaaS deployment instuctions
* Includes instructions for installing CLI

* Shows one option for installing/testing first, then gives others afterwards if people want the choice.

* Uses curl to download your gopher image as a test image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,35 @@ It demonstrates the use of a multi-stage build (available in Docker CE 17.06+) t
...
 URL: http://localhost:8080/function/img2ansi
 ```
 
+### Testing the Function
+Now that the function is running in your FaaS environment you can test it from the command line by running:
+
+```
+$ curl -SL https://raw.githubusercontent.com/johnmccabe/faas-img2ansi/master/images/gopher.png > gopher.png
+
+$ curl localhost:8080/function/img2ansi --data-binary @gopher.png
```
Comment (johnmccabe) [inline README.md:43]: the gopher image is in the `images` dir, if you want to dump the curl and set the `@gopher` to `@images\gopher`

File: README.md [hunk 2]
```diff
@@ -18,15 +18,35 @@ It demonstrates the use of a multi-stage build (available in Docker CE 17.06+) t
...
+
+Now deploy the function as follows:
 
 ```
 # faas-cli -action deploy -image=johnmccabe/faas-img2ansi -name=img2ansi -fprocess="/usr/bin/faas-img2ansi"
 200 OK
 URL: http://localhost:8080/function/img2ansi
 ```
 
+### Testing the Function
```
Comment (johnmccabe) [inline README.md:37]: I'd bump this up to `##`

**Comment by johnmccabe** [comment]
@alexellis thanks for the feedback !!!!

**Comment by johnmccabe** [comment]
Apologies for the mergeup conflict, I'd tweaked some stuff on my phone and didn't notice the PR.

File: README.md (hunk 1 above)
Comment (alexellis) [reply README.md:43]: But you don't have to clone this repo to deploy the function/image. That was my thinking.

**Comment by alexellis** [comment]
I don't need to show up as a contributor.. but if you can implement the ideas that would be fine too.

File: README.md (hunk 1 above)
Comment (johnmccabe) [reply README.md:43]: ah yeah good point !!

**Comment by johnmccabe** [comment]
No worries, I've pushed the rebase to you branch and am 👍 to :shipit: , thanks for the input !

**Pull request merged**